### PR TITLE
Supporting different devices

### PR DESCRIPTION
**Pull request by jonholmberg**
Changed the layout in the main activity so that it also contains a headline and a country flag. Added danish support. Changed the font size of the `DisplayMessageActivity` only when on landscape, just to see the difference :)

### CODE REVIEW

**Review by tudorgk — APPROVED** [review]
Can't find anything to comment on 😄 . Well done! You can merge it now.

File: jlh/JonFirstApp/app/src/main/res/layout/activity_display_message.xml [hunk 1]
```diff
@@ -10,5 +10,5 @@
     <TextView android:id="@+id/display_message"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:textSize="40dp"/>
+        android:textSize="40sp"/>
```
Comment (AndersHqst) [inline jlh/JonFirstApp/app/src/main/res/layout/activity_display_message.xml:13]: yep, nice one

**Comment by AndersHqst** [comment]
👍 looks good to me too

**Comment by jonholmberg** [comment]
Cool, thanks :) I'll merge them then.

**Pull request merged**